provisioning/roles/geerlingguy.drupal/tasks/build-composer-project.yml

```diff
@@ -1,4 +1,20 @@
 ---
+- name: Delete old /tmp/drupal-project
+  file:
+    path: "/tmp/drupal-project"
+    state: absent
+  when: not drupal_site_exists
```

Comment:
As long as the .git directory's not removed, I think this is safe enough. Worst case someone deletes index.php, runs this, finds site in a weird state, then has to go in and `git reset --hard` then probably a `composer install` by hand.

Maybe I'll regret these words someday... but I find that a pretty edge-casey scenario.